# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../e2fsprogs/no_huge_file.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./misc/mke2fs.conf.orig	2009-02-26 15:10:59.000000000 -0300
+++ ./misc/mke2fs.conf	2009-02-26 15:41:48.000000000 -0300
@@ -9,11 +9,11 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extents,huge_file,flex_bg,uninit_bg,dir_nlink,extra_isize
+		features = has_journal,extents,flex_bg,uninit_bg,dir_nlink,extra_isize
 		inode_size = 256
 	}
 	ext4dev = {
-		features = has_journal,extents,huge_file,flex_bg,uninit_bg,dir_nlink,extra_isize
+		features = has_journal,extents,flex_bg,uninit_bg,dir_nlink,extra_isize
 		inode_size = 256
 		options = test_fs=1
 	}
